package.json

```
@@ -112,7 +112,7 @@
     "mocha": "^10.2.0",
     "mongodb": "^6.9.0",
     "mongodb-runner": "^5.7.0",
-    "node-gyp": "^9.0.0",
+    "node-gyp": "^10.2.0",
```

Idk if this will work out in CI, but if you use

```suggestion
    "node-gyp": "^9.0.0 || ^10.2.0",
```

you could probably replace the `npm run replace-package` bit with just `npm install node-gyp@9` without breaking CI